File: Python/Demo7.py
class Emp:
    def __init__(self, name, age, salary, desig):
        self.name=name;
        self.age=age;
        self.salary=salary;
        self.desig=desig;

    def display(self):
        print("My name is: ", self.name, "Age is: ",self.age);

class Clerk(Emp):
    def __init__(self,name, age, salary, desig):
        Emp.__init__(self, name, age, salary, desig)

c=Clerk("Ram", 22, 30000, "Clerk");
c.display();

File: Demo7.java
import java.util.Scanner;
class Emp{
String name;
String desig;
Emp(){
Scanner sc= new Scanner(System.in);
System.out.println("Name: ");
name= sc.next();
System.out.println("Desig: ");
desig= sc.next();
}
}

class clerk extends Emp{
clerk(){
void display(){
System.out.println("Name: "+name);
System.out.println("Desig: "+desig);
}
}
}
class dev extends Emp{
dev(){
void display(){
System.out.println("Name: "+name);
System.out.println("Desig: "+desig);
}
}
}
class tester extends Emp{
tester(){
void display(){
System.out.println("Name: "+name);
System.out.println("Desig: "+desig);
}
}
}
class Demo7{
public static void main(String args[]){
clerk c= new clerk();
c.display();

dev d= new dev();
d.display();

tester t= new tester();
t.display();
}
}